irohad/ordering/impl/on_demand_ordering_service_impl.cpp

```diff
@@ -80,6 +85,10 @@ OnDemandOrderingServiceImpl::onRequestProposal(consensus::Round round) {
   // read lock
   std::shared_lock<std::shared_timed_mutex> guard(lock_);
   auto proposal = proposal_map_.find(round);
+  log_->debug("onRequestProposal, round[{}, {}], {}returning a proposal.",
```

Comment:
Seems that missed space between `{}` and `returning`, isn't it?

Reply:
No, it is not missing.